contrib/drivers/ceph/ceph_metrics.go

```diff
@@ -101,9 +101,12 @@ func (d *MetricDriver) ValidateMetricsSupportList(metricList []string, resourceT
 //	metricsList-> posted metric list
 //	instanceID -> posted instanceID
 //	metricArray	-> the array of metrics to be returned
-func (d *MetricDriver) CollectMetrics(metricsList []string, instanceID string) ([]*model.MetricSpec, error) {
+func (d *MetricDriver) CollectMetrics() ([]*model.MetricSpec, error) {
 
 	//validate metric support list
+	// Todo: Remove this two lines when Ceph driver implements modified driver interface
+	metricsList := []string{"pool_used_bytes", "pool_raw_used_bytes", "pool_available_bytes", "pool_objects_total", "pool_dirty_objects_total", "pool_read_total", "pool_read_bytes_total", "pool_write_total", "pool_write_bytes_total"}
+	instanceID := "pool1"
```

Comment:
Why is this hard-coded to "pool1"?

Reply:
This is a substitute to make the build  successful after driver interface change. C2C team will update this code in comply with new driver interface .